docker-example/Dockerfile

```diff
@@ -1,49 +1,31 @@
-FROM python:3.11-slim as base
-
-RUN adduser --disabled-password reflex
-
-
-FROM base as build
-
+# Stage 1: init
+FROM python:3.11 as init
+# Pass `--build-arg API_URL=http://app.example.com:8000` during build
+ARG API_URL
+# Copy local context to `/app` inside container (see .dockerignore)
 WORKDIR /app
-ENV VIRTUAL_ENV=/app/venv
-RUN python3 -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
 COPY . .
```

Comment:
Do we need to for example exclude anything in the COPY (like the .web directory?)

Reply:
No, in fact, `.web` is in the `.dockerignore` file because we specifically do NOT want to copy the potentially huge `.web` directory into the build context. 

This `COPY . .` command will take everything in the same directory of the `Dockerfile`, except stuff in `.dockerignore`, so it should sweep up everything needed to build the app inside the container, which is going to be better, for example if the node_modules include compiled code and the build system and container architecture differ.